components/atoms/Avatar/avatar-hover-card.tsx

```diff
@@ -4,14 +4,16 @@ import Link from "next/link";
 
 import HoverCardWrapper from "components/molecules/HoverCardWrapper/hover-card-wrapper";
 
-import { getAvatarLink } from "lib/utils/github";
+import { getAvatarByUsername } from "lib/utils/github";
+import roundedImage from "lib/utils/roundedImages";
 
 export declare interface AvatarProps {
   contributor: string;
   repositories: number[];
 }
 
 const AvatarHoverCard = ({ contributor, repositories }: AvatarProps): JSX.Element => {
+  const avatar = roundedImage(getAvatarByUsername(contributor), process.env.NEXT_PUBLIC_CLOUD_NAME);
```

Comment:
Maybe a separate issue, but we should default the `cloudName` argument to the `process.env.NEXT_PUBLIC_CLOUD_NAME` so we don't have to pass it around everywhere.

```suggestion
  const avatar = roundedImage(getAvatarByUsername(contributor));
```

Reply:
Yeah, totally. Let me update it :rocket: